### PR TITLE
DataFlow: Add `uniqueParameterNodePositionExclude`

### PR DESCRIPTION
**Pull request by RasmusWL**
When the two new consistency checks where added in https://github.com/github/codeql/pull/11564 it was not possible to exclude any nodes from these checks.

Due to the way the new Python call graph violates these checks currently, we need a way to exclude some results while figuring out how to avoid violating these. Since some of the violations are outside the test folder, it's not just possible to accept the (massive) changes to the .expected files, since the results would depend on the locations of your local Python installation.

TL;DR; I would like to have these available as a short term workaround for Python.

### CODE REVIEW

**Review by MathiasVP — APPROVED** [review]
C++/Swift 👍

**Review by atorralba — APPROVED** [review]
Java 👍

**Review by michaelnebel — APPROVED** [review]
C# LGTM

**Review by yoff — APPROVED** [review]
Python 👍
